patches/api/0446-ChunkStatusChangeEvent.patch

```diff
@@ -0,0 +1,71 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cryptite <cryptite@gmail.com>
+Date: Wed, 8 Nov 2023 09:23:23 -0600
+Subject: [PATCH] ChunkStatusChangeEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/chunk/ChunkStatusChangeEvent.java b/src/main/java/io/papermc/paper/chunk/ChunkStatusChangeEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..7d446b6219f425aabdb6a7fbb587f89e8a0f23d8
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/chunk/ChunkStatusChangeEvent.java
```

Comment:
wrong package. you are missing a /event. i think this could go into io.papermc.paper.event.world or a sub-package of that